src/logion/model/child.ts

```diff
@@ -65,21 +65,23 @@ export async function saveChildren<T extends Child>(parameters: Parameters<T>) {
     }
 }
 
-export function deleteChild<T extends Child>(childToDeleteIndex: number, children: T[], childrenToDelete: T[]) {
-    const childToDelete = children[childToDeleteIndex];
+export function deleteChild<T extends Child>(arrayIndex: number, children: T[], childrenToDelete: T[]) {
```

Comment:
Did you check that callers are indeed passing in the array index and not the DB index?

Reply:
Yes.